src/yaml/kergelen/parkings-on-slope.yaml

```diff
@@ -0,0 +1,86 @@
+group: kergelen
+name: parkings-on-slope
+version: "1.0"
+subfolder: 700-transit
+info:
+  summary: Parkings on-slope
+  description: |
+    This is a set of parkings to fit on the slopes.
+    You can use on slopes between 0 to 30 meters.
+
+    You will find the Lots under the "Rail Menu".
+    The set also includes a small add-on with some plaza pieces.
+
+    You can find the lots in the LotEditor under the name: KRG_1x1_RailSide_...
+    Important: In order to place correctly the lots need to be a road/rail above and below the slope as shown in the pictures!
+
+    CONTENTS
+     - 1 straight parking piece with tree
+     - 1 straight parking piece with lamp
+     - 1 end left parking piece
+     - 1 end right parking piece
+     - 1 plaza corner right
+     - 1 plaza corner left
+
+  author: kergelen
+  website: https://community.simtropolis.com/files/file/28905-krg-parkings-on-slope/
+  images:
+    - https://www.simtropolis.com/objects/screens/monthly_08_2013/5417a8c4ce46b336c68cbe27dd7ca6b9-parking_stex_2.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_08_2013/88aa9e66f0ca7cd3081f9a361ba40acd-parking-and-plaza-on-slope.png
+    - https://www.simtropolis.com/objects/screens/monthly_08_2013/00e53b138206443c0120fe79f884153c-parking-grid.png
+
+dependencies:
+  - mattb325:cruise-ship-terminal
+  - bsc:mega-props-newmaninc-vol02
+  - shk:parking-pack
+  - paeng:streetside-diagonal-parking
+  - bsc:mega-props-cp-vol01
+  - girafe:ashes
+  - bsc:mega-props-swi21-vol01
+  - bsc:mega-props-sg-vol01
+  - namspopof:props-pack-vol2
+assets:
+  - assetId: kergelen-parkings-on-slope
+    exclude:
+      - Kergelen Parking on slope/Add-on/
+  - assetId: mattb325-cruise-ship-terminal
+    include:
+      - Mattb325 Cruise Ship Terminal/Mattb325_RetainingWall-0x5ad0e817_0x9d7c94d6_0x30000.SC4Model
+      - Mattb325 Cruise Ship Terminal/Mattb325_RetainingWall-0x6534284a-0x13a0bd51-0xbdaab9ca.SC4Desc
+  - assetId: paeng-streetside-diagonal-parking
+    include:
+      - KEEP_SSDP_Resource.dat
```

Comment:
If you include files from the assets, you don't need the dependencies, no? Which ones do you want to keep?

Reply:
Yeah, that was the one why I asked #40. I still have to change it and split up that cruise ship terminal package into a model/prop file and a lot file.